server/routes/curfew.js

```diff
@@ -91,7 +91,16 @@ module.exports =
         const data = getIn(res.locals.licence, ['licence', 'curfew', formName]) || {}
         const { nextPath } = formConfig[formName]
 
-        res.render(`curfew/${formName}`, { bookingId, data, proposedAddress, nextPath, action })
+        const curfewAddressReviewVersion = licenceService.getCurfewAddressReviewVersion(res.locals.licence.licence)
+
+        res.render(`curfew/${formName}`, {
+          bookingId,
+          data,
+          proposedAddress,
+          nextPath,
+          action,
+          curfewAddressReviewVersion,
```

Comment:
curfewAddressReviewVersion dictates the question text to be used in the curfew address review inout page

Reply:
Could it be better to follow the same approach as risk management for consistency and show a different pug template depending on the version?

Reply:
see revisions below

Reply:
changed